src/types/TaxForm.ts

```diff
@@ -223,10 +223,29 @@ Nárok na daňový bonus (na jedno dieťa alebo úhrn na viac vyživovaných det
   ziadamVratitDanovyPreplatok: boolean
   iban: string
 
+  /** Summary values - all in one place - some duplicated **/
+  summary: {
```

Comment:
@michal-rohacek toto by som nedaval sem, lebo to nepatri do danoveho priznania. `TaxForm` ma obsahovat iba data ktore sa vyplnaju na financnej sprave. Toto by mohol byt samostatny interface a buildovalo by sa to samostatnou metodou - kludne v `calculation.ts` ale nie v `calculate()`.